sbt-scalajs-bundler/src/main/scala/scalajsbundler/sbtplugin/ScalaJSBundlerPlugin.scala

```diff
@@ -348,6 +348,18 @@ object ScalaJSBundlerPlugin extends AutoPlugin {
       "Custom arguments to webpack"
     )
 
+    /**
+      * node.js cli flags https://nodejs.org/api/cli.html
+      *
+      * Defaults to an empty list.
+      *
+      * @group settings
+      */
+    val webpackNodeFlags = SettingKey[Seq[String]](
+      "webpackNodeFlags",
+      "Custom arguments to webpack"
```

Comment:
The description is not correct: it’s arguments to the `node` program when webpack is run.